css/styles.css

```diff
@@ -137,6 +137,25 @@ nav>ul {
 
 /* Three lines stacked will create hamburger icon */
 
+<!-- Ingredients
+1 1/2 pounds 80% lean 20% fat ground beef or ground chuck
+1 tablespoon Worcestershire sauce
+1 1/2 teaspoons seasoning salt
+1 teaspoon garlic powder
+1/2 teaspoon ground black pepper
+Optional: 4 slices of cheese
+4 hamburger buns
+Optional: hamburger toppings - lettuce tomato, onion, pickles, ketchup, mustard, mayo, etc.
+Instructions
+Preheat the grill to 375 degrees F (medium-high).
+In a large bowl, add the beef. Sprinkle evenly with the Worcestershire sauce, seasoning salt, garlic powder, and pepper. Use your hands to mix the ingredients until they are just combined.
+Divide the meat mixture into fourths. Take 1/4 of the meat mixture and use your hands to press it into the shape of a hamburger patty that is about 3/4 inch thick. Make an indention in the middle of the patty to prevent bulging in the center of the hamburger as it cooks. Repeat with the remaining meat mixture, making 4 hamburgers.
+Place the burgers on the grill. Cook 4-5 minutes on the first side. Flip the burgers over and cook an additional 4-5 minutes, until the burgers have reached the desired doneness.*
```

Comment:
`Make an indention in the middle of the patty **to prevent bulging** in the center of the hamburger as it cooks`
? I always beat my meat to prevent bulging, this recipe makes no sense and change should not be added

Reply:
ok